src/screens/AboutApp/styles.js

```diff
@@ -0,0 +1,77 @@
+import { StyleSheet } from "react-native";
+import { black87, black06, blue, gray } from "../../constants/colors";
+
+const styles = StyleSheet.create({
+  container: {
+    margin: 20
+  },
+  logo: {
+    flex: 1,
+    alignItems: "center"
+  },
+  image: {
+    marginBottom: 20,
+    width: 185,
+    height: 100
+  },
+  version: {
+    fontSize: 12
+  },
+  buttonWrapper: {
+    marginTop: 20
+  },
+  text: {
+    marginTop: 10,
+    color: black87,
+    fontFamily: "Roboto",
```

Comment:
why do we use different fonts on the same screen? is it on purpose or just designer mistake?

Reply:
designer use two different fonts in our layout